lib/srv/alpnproxy/auth/auth_proxy_test.go

```diff
@@ -0,0 +1,56 @@
+package alpnproxyauth
+
+import (
+	"context"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/services"
+	"github.com/stretchr/testify/require"
+	"net"
+	"testing"
+)
+
+type mockAuthGetter struct {
+	servers []types.Server
+}
+
+func (m mockAuthGetter) GetClusterName(...services.MarshalOption) (types.ClusterName, error) {
+	return nil, nil
+}
+
+func (m mockAuthGetter) GetAuthServers() ([]types.Server, error) {
+	return m.servers, nil
+}
+
+func TestDialLocalAuthServerNoServers(t *testing.T) {
+	s := NewAuthProxyDialerService(nil, mockAuthGetter{servers: []types.Server{}})
+	_, err := s.dialLocalAuthServer(context.Background())
+	require.Error(t, err)
+	require.Equal(t, "empty auth servers list", err.Error())
```

Comment:
As Edoardo mentioned below, I'd go with a Contains and match just enough of the message to be satisfied.

Reply:
Missed this one?

Reply:
It has the comment now, maybe something was not refreshed earlier?